samples/test_validation.py

```diff
@@ -12,14 +12,12 @@ def validate_document_with_schema(document, schema):
 
     except ValidationError as e:
         logging.error(
-            "Schema Validation Error! [{}] does not validate against schema. Error [{}]".format(
-                document, e
-            )
+            f"Schema Validation Error! [{document}] does not validate against schema. Error [{e}]"
```

Comment:
does this allow you to write logs like this ("foo {} bar {}", var1, var2)

Reply:
Probably should deicide a process on how we log

Reply:
can do `"Schema Validation Error! %s does not validate against schema. Error %s", document, e`

Reply:
We are using the f string for most of the logs